### PR TITLE
Split rpc calls

### PR DESCRIPTION
**Pull request by eesmerdag**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Example:
 ```json
 {
   "app_env": "production",
-  "rpc_endpoint": "http://YOUR_NODE_RPC_IP:PORT",
+  "rpc_endpoints": "http://YOUR_NODE_RPC_IP:PORT,http://YOUR_NODE_RPC_IP2:PORT2",
```
Comment (sosedoff) [inline README.md:75]: why comma separated values when we have a luxury or having an array or strings?

**Review by sosedoff — COMMENTED** [review]
Looks like we have a bunch of duplicated code around, its not a big deal, but we could improve on the concept and introduce a `ClientPool` to manage a set of client connections. That way you implement it once and pass it around just like you would with the current client. Or, we can implement multiple endpoints support in the client itself. Just thinking out loud here.

**Review by pacmessica — COMMENTED** [review]
BP added lib code that achieves this, see https://github.com/figment-networks/indexing-engine/tree/master/worker

**Comment by sosedoff** [comment]
@pacmessica this is a slightly different setup - basically we cant use DH API as is (because of throttling) so we round robin requests using our own code. if we were to use the worker code you're referring to, do we need to run an additional process on each node?

**Comment by pacmessica** [comment]
> @pacmessica this is a slightly different setup - basically we cant use DH API as is (because of throttling) so we round robin requests using our own code. if we were to use the worker code you're referring to, do we need to run an additional process on each node?

ah yeah true, I believe we'd need to run additional worker services

**Pull request merged**